test/default.env

```diff
@@ -1,13 +1,17 @@
-DOCKER_WIKIBASE_REPO_NAME=test-wikibase-1
-DOCKER_MYSQL_NAME=test-mysql-1
+## DEFAULT Test Environment Setup
+# These values override values set from `../example/template.env`
```

Comment:
Maybe even the file name could indicate that this is an env override

Reply:
Agree that the naming could be improved. I would prefer simply call this `test.env` as it will contain mostly test specific values vs overrides (which should be minimised).